Server/EVESharp.Node/Services/Graphics/beyonce.cs

```diff
@@ -0,0 +1,107 @@
+using System;
+using EVESharp.EVE.Network.Services;
+using EVESharp.EVE.Network.Services.Validators;
+using EVESharp.EVE.Sessions;
+using EVESharp.Types;
+using EVESharp.Types.Collections;
+
+namespace EVESharp.Node.Services.Graphics
+{
+    // This must match the service name the client moniker uses.
+    [ConcreteService("beyonce")]
+    public class beyonce : ClientBoundService
+    {
+        public override AccessLevel AccessLevel => AccessLevel.None;
+
+        // Global (unbound) service instance – constructed at startup
+        public beyonce(IBoundServiceManager manager)
+            : base(manager)
+        {
+            Console.WriteLine("[beyonce] global service constructed");
+        }
+
+        // Per-client bound instance
+        private beyonce(IBoundServiceManager manager, Session session, int objectID)
+            : base(manager, session, objectID)
+        {
+            Console.WriteLine(
+                $"[beyonce] bound instance created: objectID={objectID}, charID={session.CharacterID}"
+            );
+        }
+
+        // --- Binding logic ----------------------------------------------------
+
+        /// <summary>
+        /// Create the bound instance. The client moniker for beyonce uses
+        /// (\"beyonce\", solarSystemID), so bindParams.ObjectID == solarSystemID.
+        /// </summary>
+        protected override BoundService CreateBoundInstance(ServiceCall call, ServiceBindParams bindParams)
+        {
+            int objectID = bindParams.ObjectID;
+
+            Console.WriteLine($"[beyonce] CreateBoundInstance for solarSystemID={objectID}");
+            return new beyonce(BoundServiceManager, call.Session, objectID);
+        }
+
+        /// <summary>
+        /// Decide which node should host this object. For now, always local.
+        /// </summary>
+        [MustBeCharacter]
+        protected override long MachoResolveObject(ServiceCall call, ServiceBindParams bindParams)
+        {
+            Console.WriteLine("[beyonce] MachoResolveObject -> local node");
+            return call.MachoNet.NodeID;
```

Comment:
This should actually return what node has this solar system booted. If it's not booted anywhere it should be loaded somewhere in here, so that NodeID should be returned

Reply:
For example, dogmaIM does this:
https://github.com/Almamu/EVESharp/blob/b19fc60f7b6f3a6c3c1914835d8b42c92014cf8b/Server/EVESharp.Node/Services/Dogma/dogmaIM.cs#L240
https://github.com/Almamu/EVESharp/blob/b19fc60f7b6f3a6c3c1914835d8b42c92014cf8b/Server/EVESharp.Node/Services/Dogma/dogmaIM.cs#L250